content/en/agent/kubernetes/control_plane.md

```diff
@@ -598,6 +599,181 @@ The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches
+  namespace: cattle-monitoring-system
+  labels:
+    component: kube-scheduler
+    k8s-app: pushprox-kube-scheduler-client
+  annotations:
+    ad.datadoghq.com/endpoints.check_names: '["kube_scheduler"]'
+    ad.datadoghq.com/endpoints.init_configs: '[{}]'
+    ad.datadoghq.com/endpoints.instances: |
+      [
+        {
+          "prometheus_url": "http://%%host%%:10251/metrics"
+        }
+      ]
+spec:
+  clusterIP: None
+  selector:
+    k8s-app: pushprox-kube-scheduler-client
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pushprox-kube-controller-manager-datadog
+  namespace: cattle-monitoring-system
+  labels:
+    component: kube-controller-manager
+    k8s-app: pushprox-kube-controller-manager-client
+  annotations:
+    ad.datadoghq.com/endpoints.check_names: '["kube_controller_manager"]'
+    ad.datadoghq.com/endpoints.init_configs: '[{}]'
+    ad.datadoghq.com/endpoints.instances: |
+      [
+        {
+          "prometheus_url": "http://%%host%%:10252/metrics"
+        }
+      ]
+spec:
+  clusterIP: None
+  selector:
+    k8s-app: pushprox-kube-controller-manager-client
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pushprox-kube-etcd-datadog
+  namespace: cattle-monitoring-system
+  labels:
+    component: kube-etcd
+    k8s-app: pushprox-kube-etcd-client
+  annotations:
+    ad.datadoghq.com/endpoints.check_names: '["etcd"]'
+    ad.datadoghq.com/endpoints.init_configs: '[{}]'
+    ad.datadoghq.com/endpoints.instances: |
+      [
+        {
+          "prometheus_url": "https://%%host%%:2379/metrics",
+          "tls_ca_cert": "/host/opt/rke/etc/kubernetes/ssl/kube-ca.pem",
+          "tls_cert": "/host/opt/rke/etc/kubernetes/ssl/kube-etcd-172-29-160-41.pem",
+          "tls_private_key": "/host/opt/rke/etc/kubernetes/ssl/kube-etcd-172-29-160-41-key.pem"
+        }
+      ]
+spec:
+  clusterIP: None
+  selector:
+    k8s-app: pushprox-kube-etcd-client
+```
+
+Deploy the Datadog Agent with manifests based on the following configurations:
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+Custom `values.yaml`:
+
+```
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  kubelet:
+    tlsVerify: false
+agents:
+  volumes:
+    - hostPath:
+        path: /opt/rke/etc/kubernetes/ssl
+      name: etcd-certs
+  volumeMounts:
+    - name: etcd-certs
+      mountPath: /host/opt/rke/etc/kubernetes/ssl
+      readOnly: true
+  tolerations:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/controlplane
+    operator: Exists
+  - effect: NoExecute
+    key: node-role.kubernetes.io/etcd
+    operator: Exists
+```
+
+{{% /tab %}}
+{{% tab "Operator" %}}
+
+DatadogAgent Kubernetes Resource:
+
+```
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  agent:
+    image:
+      name: "gcr.io/datadoghq/agent:latest"
```

Comment:
thanks to operator v0.7.0 and the defaulting logic. I think we can remove the need of defining the image

```suggestion
```